cisco_sdwan/assets/monitors/device_reboot.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 2,
+  "created_at": "2024-04-05",
+  "last_updated_at": "2024-04-05",
+  "title": "Cisco SD-WAN Device has rebooted several times",
+  "tags": [
+    "integration:cisco-sdwan"
+  ],
+  "description": "This monitor detects instances of repeated reboots, serving as an indicator for potential issues like software-related glitches, low memory conditions, corrupt software images, hardware faults, or software bugs. By identifying these concerns, it ensures network stability and optimized performance.",
```

Non blocking optional suggestion. Thanks for adding the impact!
```suggestion
  "description": "Repeated reboots are an indication of potential issues like software-related glitches, low memory conditions, corrupt software images, hardware faults, or software bugs. By identifying these concerns, this monitor ensures network stability and optimized performance.",
```